esm-samples/jsapi-angular-cli/tsconfig.json

```diff
@@ -16,12 +16,13 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2020",
+    "target": "ES2022",
     "module": "es2020",
     "lib": [
       "es2020",
       "dom"
-    ]
+    ],
+    "useDefineForClassFields": false
```

Comment:
- should the lib be updated too match the target?
- do we need to explicitly set `useDefineForClassFields`, or is this to benefit older versions?